src/web/components/App.tsx

```diff
@@ -399,6 +400,19 @@ export const App = ({ CAPI, NAV }: Props) => {
         CAPI.config.isSensitive,
     ]);
 
+    // ************************
+    // *   Google Analytics   *
+    // ************************
+    useEffect(() => {
+        onConsentChange((state: any) => {
+            const consentGiven = getConsentFor('google-analytics', state);
+            if (consentGiven) {
+                loadScript('https://www.google-analytics.com/analytics.js');
+                loadScript(window.guardian.gaPath);
+            }
```

Comment:
I think we ought to have an `else` condition here that sets `window.ga` to a no-op like we have done in the frontend PR. We should stop sending any data to GA as soon as someone removes their consent.

Reply:
As we have it setup now, if the reader removes consent then any subsequent _page views_ will not send data to GA but if, as you suggest, we set `window.ga = () => {}` in the else then this should block _all_ future calls, which is much better.

Sematically, it might be nice to use `() => false`? This will also make testing for this use case clearer